Cargo.toml

```diff
@@ -13,3 +13,6 @@ members = ["rust/scx_stats",
            "scheds/rust/scx_layered",
            "scheds/rust/scx_mitosis"]
 resolver = "2"
+
+[profile.release]
```

Comment:
Maybe it's better to add another [profile instead](https://doc.rust-lang.org/cargo/reference/profiles.html#custom-profiles):
```
[profile.release-lto]
inherits = "release"
lto = true
```

Reply:
Personally I would prefer, that the default would use LTO, and for the CI we pass LTO off or so.

Reply:
The meson flag can be added, then we activate this in PKGBUILD.